apps/test/unit/templates/courseOverview/CourseOverviewTest.js

```diff
@@ -168,68 +169,25 @@ describe('CourseOverview', () => {
     });
 
     it('appears when two versions are present and viewable', () => {
-      const versions = [
-        {
-          name: 'csp-2017',
-          year: '2017',
-          title: '2017',
-          canViewVersion: true,
-          isStable: true,
-          locales: [],
-          localeCodes: []
-        },
-        {
-          name: 'csp-2018',
-          year: '2018',
-          title: '2018',
-          canViewVersion: true,
-          isStable: true,
-          locales: [],
-          localeCodes: []
-        }
-      ];
       const wrapper = shallow(
         <CourseOverview
           {...defaultProps}
-          versions={versions}
+          versions={courseOfferings['2'].course_versions}
```

Comment:
These indices are a bit confusing to me. How did you decide which course offering to use?

Reply:
Different courseOfferings have different properties I was trying to test out. Some have one course_version some have multiple. Some have content roots that are Script and some are UnitGroups. So for this one I wanted something with two course_versions so I used the CSD course offering test data